.wp-env.json

```diff
@@ -1,5 +1,5 @@
 {
-  "core": null,
+  "core": "https://wordpress.org/wordpress-6.8-beta1.zip",
```

Comment:
@joemcgill @felixarntz Without WordPress 6.8, I can't traverse the context since we fixed that issue in Core for 6.8. So, I have to use version 6.8 here.

Reply:
Yes. Updated in https://github.com/WordPress/performance/pull/1795/commits/b981edc9c2038cff26957ea17599cfc6308ade78

Just for information. Once 6.8 will release we update plugin minimum WP version to 6.8 and revert these changes so it will not affect other plugins.

Reply:
Maybe `6.8-RC1` would be better?

Reply:
For now, let's leave it as trunk. Since WordPress 6.8 will be released in a few weeks, we'll need to revert these changes anyway.